rj_gameplay/stp/role/assignment/naive.py

```diff
@@ -14,6 +14,7 @@
 SortedRequests = List[assignment.FlatRoleRequests]
 
 # Define some big constant for "hard constraints".
+# TODO: Make a local param
```

Comment:
This doesn't need to be a local param, it shouldn't affect behavior.

Reply:
I disagree, it might be more useful to have the INVALID_COST number accessible to the unassigned cost_fn methods. For example a goalie that's unassigned should give the same massive cost

Reply:
Alternatively, we can fix role assignment to handle `float('inf')` as this invalid cost/"big number"

Reply:
I mean role assignment can handle float('inf'), it actually purposefully, manually throws an error. However, unassigned roles are given a cost of infinity in my implementation to somehow make sure they never get assigned, since all other costs should be finite.